ASSETS.md

```diff
@@ -38,6 +42,8 @@ requirement:
 
 The definition of the manifest is not complete yet and further details will follow.
 
+Question: How do we handle if only one metricset is in beta but integration is in GA?
```

Comment:
@exekias Not to be answered in this PR, but interested to hear your thoughts.

Reply:
that's a good question, I guess we should not add a beta metricset to defaults? It could also be interesting to flag it in the provided configuration (or even the UI)

Reply:
Good idea with the defaults. So it's documented with the specific input that it's in beta so it's going to show up in the UI when the user wants to add it.